api/v1/routes/blog.py

```diff
@@ -0,0 +1,53 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from sqlalchemy.orm import Session
+from api.db.database import get_db
+from api.v1.models.blog import Blog
+from api.v1.schemas.blog import DeleteBlogResponseSchema
+from api.utils.dependencies import get_current_admin
+import logging
+import uuid
+
+
+blog = APIRouter(prefix="/api/v1/blogs", tags=["Blogs"])
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
```

Comment:
You have made a mistake with the `prefix`, it should be `/blogs` only

Also you are like the 3rd person to implement logging you guys should implement it properly. You can turn it to a utility for everyone to use

Reply:
I have removed the logger